jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarProcessor.java

```diff
@@ -145,6 +149,42 @@ public static JarType determineJarType(Path jarPath) throws IOException {
     return layers;
   }
 
+  /**
+   * Compute the entrypoint for a standard jar in exploded mode.
+   *
+   * @param jarPath path to jar file
+   * @param tempDirPath path to temporary jib local directory
+   * @param layers list of {@link FileEntriesLayer}
+   * @return list of {@link String} representing entrypoint
+   * @throws IOException if I/O error occurs when opening the jar file or if temporary directory
+   *     provided doesn't exist
+   * @throws IllegalArgumentException if main class is not found in the jar manifest
+   */
+  public static ImmutableList<String> computeEntrypoint_explodedStandard(
+      Path jarPath, Path tempDirPath, List<FileEntriesLayer> layers)
+      throws IOException, IllegalStateException {
```

Comment:
I admit we don't have a good established policy around throwing, handling, and reporting exceptions and errors. We have not done well so far. But at least the situation for Jib CLI is not so grim, as it is not a library but a CLI. So I think throwing a runtime exception to crash for this type of errors is more or less acceptable.

Anyways, what I wanted to say is that, I think it's fine to not specify `IllegalStateException` (or `IllegalArgumentException`). Also in the Javadoc.